Framework/README.md

```diff
@@ -34,8 +34,8 @@ The goal of this UI framework is to identify library and framework sets, provide
 - Asynchronous data fetching (Ajax) and bi-directional socket (WebSockets)
 - MVC engine with a "diffing" algorithm
 
-### System requirements
-* `nodejs` >= 8.9.4
+### Backend requirements
+* `nodejs` >= v10.13.0
```

Comment:
This will be true only for the newer versions of the framework. Should we specify that we still support `nodejs >= 8.9.4` for the framework up to version 1.9.4?

Reply:
I think we want users to use the latest version